lua/plugins/core.lua

```diff
@@ -0,0 +1,131 @@
+return {
```

Comment:
Thoughts on bringing `nvim-ts-rainbow` since there is a maintained fork?

Reply:
https://github.com/mrjones2014/nvim-ts-rainbow

Reply:
@jay-babu has this been decided? There was discussion on whether or not this was going to move to a new organization as there were two people interested in maintaining and fully rewriting the plugin. Both currently have maintained forks and were going to discuss merging them. I will not be adding it until we can prove which fork will be maintained.

Reply:
Oh, unsure.

https://www.reddit.com/r/neovim/comments/1079zgs/this_week_in_neovim_26_mon_jan_9th_2022/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

Was simply going of this.